indico/modules/events/papers/templates/management/_paper_assignment_list.html

```diff
@@ -0,0 +1,115 @@
+                                <span class="vertical-aligner">
+                                    {# TODO: Link to the paper timeline page #}
+                                    <a href="#">
+                                        {{- contrib.title -}}
+                                    </a>
+                                </span>
+                            </td>
+                            <td class="i-table state-column" data-searchable="{{ last_revision_state | lower }}">
+                                <span class="vertical-aligner">{{ last_revision_state }}</span>
+                            </td>
+                            <td class="i-table revision-column">
+                                <span class="vertical-aligner">
+                                    {% if contrib.paper_revisions %}
+                                        {{ contrib.paper_revisions | length }}
+                                    {% endif %}
+                                </span>
+                            </td>
+                            <td class="i-table person-row-cell"
+                                data-searchable="{{ contrib.paper_judges|map(attribute='name')|join(', ')|lower }}">
+                                <span class="vertical-aligner">
+                                    {% for judge in contrib.paper_judges -%}
+                                        <div class="person-row icon-user">{{ judge.display_full_name }}</div>
+                                    {%- endfor %}
+                                </span>
+                            </td>
+                            <td class="i-table person-row-cell"
+                                data-searchable="{{ contrib.paper_content_reviewers|map(attribute='name')|join(', ')|lower }}">
+                                <span class="vertical-aligner">
+                                    {% for reviewer in contrib.paper_content_reviewers -%}
+                                        <div class="person-row icon-user">{{ reviewer.display_full_name }}</div>
+                                    {%- endfor %}
+                                </span>
+                            </td>
+                            <td class="i-table person-row-cell"
+                                data-searchable="{{ contrib.paper_layout_reviewers|map(attribute='name')|join(', ')|lower }}">
+                                <span class="vertical-aligner">
+                                    {% for reviewer in contrib.paper_layout_reviewers -%}
+                                        <div class="person-row icon-user">{{ reviewer.display_full_name }}</div>
+                                    {%- endfor %}
+                                </span>
+                            </td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        </form>
+    {%- else %}
+        {%- call message_box('info') %}
```

Comment:
I would move this to be the first block. This helps see more clearly the implications of the `if` block.

```jinja
{% if not contribs %}
    {%- call message_box('info') %}
    ...
{% else %}
    <form method="POST">
    ...
{% endif %}
```